src/hotspot/share/utilities/vmError.cpp

```diff
@@ -166,6 +170,48 @@ static void print_bug_submit_message(outputStream *out, Thread *thread) {
   out->print_raw_cr("#");
 }
 
+static bool check_stack_headroom(Thread* thread,
+                                 size_t headroom) {
+  static const address stack_top    = thread != nullptr
```

Comment:
We typically call "stack_top "stack_base", and "stack_bottom" we call "stack_end".

Reply:
There is existing code in this file that uses this naming:
```
 STEP_IF("printing stack bounds", _verbose)
    st->print("Stack: ");

    address stack_top;
    size_t stack_size;

    if (_thread) {
      stack_top = _thread->stack_base();
      stack_size = _thread->stack_size();
    } else {
      stack_top = os::current_stack_base();
      stack_size = os::current_stack_size();
    }

    address stack_bottom = stack_top - stack_size;
    st->print("[" PTR_FORMAT "," PTR_FORMAT "]", p2i(stack_bottom), p2i(stack_top));
```